config/versions/v0/v0.go

```diff
@@ -0,0 +1,19 @@
+package v0
+
+import (
+	"context"
+)
+
+// Version is a baseline version with no changes, so we can downgrade back to nothing
+// It does not implement any upgrade interfaces
+type Version struct{}
+
+// UpgradeConfig is an empty stub
+func (v *Version) UpgradeConfig(_ context.Context, j []byte) ([]byte, error) {
```

Comment:
missing coverage on this file

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/340a134a8a